components/dashboard_modules/ExpectedOutput.tsx

```diff
@@ -0,0 +1,43 @@
+import NumberDisplay from './NumberDisplay';
+import { formatter, useFutureGraphData } from 'lib/utils';
+
+interface ForecastDataPoint {
+  target_datetime_utc: number;
+  expected_generation_kw: number;
+}
+
+const computeMipointRiemmanSum = (points: ForecastDataPoint[] | undefined) => {
+  if (!points) {
+    return 'Error';
+  }
+
+  var approxArea = 0.0;
+  console.log(points);
+
+  for (let i = 0; i < points.length - 1; i++) {
+    const avgHeight =
+      (points[i].expected_generation_kw +
+        points[i + 1].expected_generation_kw) /
+      2;
+    const timeDiffMili =
+      new Date(points[i + 1].target_datetime_utc).getTime() -
+      new Date(points[i].target_datetime_utc).getTime();
+
+    const timeDiffHours = timeDiffMili / (3.6 * Math.pow(10, 6));
```

Comment:
`(3.6 * Math.pow(10, 6))` is a magic number, can you separate it into a constant above the for loop or above this variable definition?

You can also just write the literal `3.6e6` instead of computing the above.

Reply:
can call it millisInHour or something